### ISSUE TITLE
GLideN64 resolution bug when exiting PJ64 with F12.

### ISSUE THREAD

**Issue by AmbientMalice** [gonetz/GLideN64#727]
Basically, GLideN64 doesn't correctly reset display resolution when exiting a game while in fullscreen using F12 with Project 64.

**Comment by gonetz** [comment]
@AmbientMalice Is it still actual?

**Comment by oddMLan** [comment]
Yes, it is still actual. Make sure that the fullscreen resolution is different than the windowed resolution. It also happens when you change the windowed resolution for one game in specfic with profiles, press F12, then boot up another game that reads from the global profile, thus a different resolution.
